pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -388,6 +391,78 @@ class OSRow extends React.Component {
     }
 }
 
+const UnattendedRow = ({ validationFailed, unattendedDisabled, unattendedInstallation, os, profile, onValueChanged }) => {
+    const validationStatePassword = validationFailed.password ? 'error' : undefined;
+    let unattendedInstallationCheckbox = (
+        <label className="checkbox-inline">
+            <input id="unattended-installation" type="checkbox"
+                checked={unattendedInstallation}
+                disabled={unattendedDisabled}
+                onChange={e => onValueChanged('unattendedInstallation', e.target.checked)} />
+            {_("Run unattended installation")}
+        </label>
+    );
+    if (unattendedDisabled) {
+        unattendedInstallationCheckbox = (
+            <OverlayTrigger overlay={ <Tooltip id='os-unattended-installation-tooltip'>{ _("The selected Operating System does not support unattended installations") }</Tooltip> } placement='left'>
+                {unattendedInstallationCheckbox}
+            </OverlayTrigger>
+        );
+    }
+
+    return (
+        <>
+            {unattendedInstallationCheckbox}
+            {!unattendedDisabled && unattendedInstallation ? <>
+                {os.profiles.length > 0 && <>
+                    <label className="control-label" htmlFor="profile-select">
+                        {_("Profile")}
+                    </label>
+                    <Select.Select id="profile-select"
+                        initial={os.profiles && os.profiles[0]}
+                        onChange={e => onValueChanged('profile', e)}>
+                        { (os.profiles || []).sort()
+                                .reverse() // Let jeos (Server) appear always first on the list since in osinfo-db it's not consistent
+                                .map(profile => {
+                                    let profileName;
+                                    if (profile == 'jeos')
+                                        profileName = 'Server';
+                                    else if (profile == 'desktop')
+                                        profileName = 'Workstation';
+                                    else
```

Comment:
So these are special cases for Fedora/RHEL? This is something that would be nice to split into a separate function, as the whole nested `<>` rendering is already hard enough to read. Also, I'm not sure if "Server"/"Workstation" should be translated, or being treated as proper nouns.

Feel free to ignore this comment! Just a minor thought.

Reply:
This is not specific to Fedora/RHEL.
libosindo has the wording of 'jeos' for all Server media and 'desktop' for Workstations. 
The sort/reverse array calls I have there are in order to get always `Server` before `Workstation` in the `Dropdown` options, regardless in what order osinfo db has the data.

Reply:
Interesting, I thought "jeos" was a particular product (incidentally, Ubuntu had a "JEOS" flavor many years ago, "just enough operating system" -- a minimal OS for VMs -- but I suppose that's not what Fedora's is)

Reply:
FWIW: SUSE has officially used "jeos" for years for small builds too. Leap and Tumbleweed both have a "JeOS" tab even today.